src/Uno.UI/UI/Xaml/Controls/Image/Image.iOS.cs

```diff
@@ -243,6 +244,7 @@ private void SetNeedsLayoutOrDisplay()
 
 		private void UpdateContentMode(Stretch stretch)
 		{
+#if !NEW_IMAGE_POSITIONNING
```

Comment:
How will it be disabled ?

Reply:
That's not the final commit...  stay tuned for this.